docs/tutorial/appendix/source-JSON-YAML/index.md

```diff
@@ -0,0 +1,274 @@
+---
+title: "Sourcing from JSON or YAML"
+---
+
+# Beforehand
+
+As you're presented with Gatsby, you'll see that you have at your disposal a myriad of ways to pull data from into your website, ranging from the most used ones like [Contenfull](https://www.contentful.com/), or [WordPress](https://wordpress.com/), [Drupal](https://www.drupal.org/), to the most edgier ones like [ButterCMS](https://buttercms.com/), [GraphCMS](https://graphcms.com/) and their appropriate plugins to ensure connectivity.
+
+With this tutorial, it will help you return to the origins and instead of pulling data in from the providers above, you'll use a plain JSON file and YAML to achieve this.
```

Comment:
Use an active introduction: "This tutorial will help you return..."

Also make it clear these are distinct options–they don't have to use both JSON and YAML. "..you'll see examples using plain JSON and YAML to achieve this"